packages/hdwallet-keepkey/src/bitcoin.ts

```diff
@@ -28,9 +28,32 @@ import {
 
 import { KeepKeyTransport } from './transport'
 
-import * as Messages from '@keepkey/device-protocol/lib/messages_pb'
-import * as Types from '@keepkey/device-protocol/lib/types_pb'
-import * as Exchange from '@keepkey/device-protocol/lib/exchange_pb'
+import {
```

Comment:
sigh. had to change all these imports because `device-protocol`'s protobuf compiler got updated, and outputs slightly different, and incompatible js modules.